src/App.js

```diff
@@ -2,16 +2,34 @@ import React, { useEffect, useState } from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
```

Comment:
Remove extra space between modules and component declaration

Reply:
Sure